src/schema/v2/Feature/FeatureType.ts

```diff
@@ -41,18 +42,26 @@ export const FeatureType = new GraphQLObjectType<
     },
     setsConnection: {
       type: OrderedSetConnection.connectionType,
-      args: pageable(),
+      args: pageable({
+        sort: {
+          type: OrderedSetSortsEnum,
+          defaultValue: OrderedSetSortsEnum.getValue("KEY"),
```

Comment:
Should this be `KEY_ASC`?

Reply:
Good catch. Wish the types inferred the valid keys :/